### ISSUE TITLE
Remove Search Button

### ISSUE THREAD

**Issue by monddelmundo** [Juggernaut9/pokemon#58]
Hi, I will be removing the Search button.

**Comment by ayushpattnaik** [comment]
close the issue

**Comment by monddelmundo** [comment]
Hi @ayushpattnaik ,

This was discussed [here](https://github.com/Juggernaut9/pokemon/pull/49#issuecomment-703317141).
Can you let me know why you want this to be closed?

Thanks!

**Comment by ayushpattnaik** [comment]
hi @monddelmundo 
I mean the functionality is already added to the search button right? why to remove then?

**Comment by monddelmundo** [comment]
Hi @ayushpattnaik ,
@Juggernaut9 said it is no longer needed since the search bar is working. He confirmed that we should just delete it instead [here](https://github.com/Juggernaut9/pokemon/pull/49#issuecomment-703317141).
Thanks!

**Comment by ayushpattnaik** [comment]
Okay great!!

**Comment by BlckShadow** [comment]
Is this still open?

**Comment by monddelmundo** [comment]
Yes @BlckShadow . Waiting for PR #59  to be reviewed and merged.

**Comment by BlckShadow** [comment]
Ok I was just wondering since I'm a beginner and I want to take a react app since I'm new to react and I want to challenge my for final PR

**Comment by BlckShadow** [comment]
@monddelmundo  Another q: Is only the button to remove and not the search field Correct? Would that make the search field useless?

**Comment by monddelmundo** [comment]
Hi @BlckShadow , No, the search field is functional even without the button. You can try it here:
https://pokemon-hacktoberf-git-a2b908049849d42f333e10ed678078cb8-daada4.juggernaut9.vercel.app

You can look for the other issues posted for this project and try to solve those, like beautifying the site (#5 ).

**Comment by BlckShadow** [comment]
@monddelmundo Ah ok that's awesome

**Comment by BlckShadow** [comment]
Ok I would love to take this on for my final PR for Digital Ocean Hacktober for my final PR

**Comment by BlckShadow** [comment]
Ok, I also submit one for PR for this issue; thanks again for the attempt and learning experience with this hack fest

**Comment by BlckShadow** [comment]
@ayushpattnaik Hi I submitted a puller request for this issue but it still pending so I'm not sure what to do

**Comment by ayushpattnaik** [comment]
> @ayushpattnaik Hi I submitted a puller request for this issue but it still pending so I'm not sure what to do

I guess the repo is inactive

**Comment by BlckShadow** [comment]
Can we message him or something

**Comment by BlckShadow** [comment]
Can I ask a question?
